src/banana.js

```diff
@@ -180,6 +192,40 @@ module.exports = function bananaChecker( dir, options, logErr ) {
 		} );
 	}
 
+	let sourceMessageLeadingWhitespace = [];
+	let documentationMessageLeadingWhitespace = [];
+	if ( !options.allowLeadingWhitespace ) {
+		sourceMessageLeadingWhitespace = sourceMessageKeys.filter(
+			function ( message ) {
+				return leadingWhitespaceRegex.test( sourceMessages[ message ] );
+			}
+		);
+
+		documentationMessageLeadingWhitespace = documentationMessageKeys.filter(
+			function ( message ) {
+				return leadingWhitespaceRegex.test( documentationMessages[ message ] );
+			}
+		);
+	}
+
+	let sourceMessageTrailingWhitespace = [];
+	let documentationMessageTrailingWhitespace = [];
+	if ( !options.allowTrailingWhitespace ) {
+		sourceMessageTrailingWhitespace = sourceMessageKeys.filter(
+			function ( message ) {
+				return trailingWhitespaceRegex.test( sourceMessages[ message ] );
+			}
+		);
+
+		documentationMessageTrailingWhitespace = documentationMessageKeys.filter(
+			function ( message ) {
+				return trailingWhitespaceRegex.test( documentationMessages[ message ] );
+			}
+		);
+	}
+
+	let sourceMessageMissing = [];
```

Comment:
Linter is complaining this has already been declared.

Reply:
How odd. It passes locally?

Reply:
Oh, it pushed the wrong HEAD. Meh. Fixed.